task_12/src/css/desktop/main.scss

```diff
@@ -0,0 +1,92 @@
+@import '../variables.scss';
+
+main {
+    padding: 40px 30px 0;
+
+    font-family: $main-font;
+}
+
+.button {
+    position: relative;
+
+    display: block;
+    overflow: visible; /* IE9 & 10 */
+
+    max-width: 200px;
+
+    cursor: pointer;
+    transition: 500ms linear;
+
+    color: $text-color;
+    border: 1px solid $text-color;
+    border-radius: 5px;
+    outline: none;
+    background-color: $hover-text-color;
+    box-shadow: inset 1px 1px 0 rgba(255, 255, 255, .25), /* highlight */
+    inset 0 0 6px #a23227, /* inner glow */
+    inset 0 80px 80px -40px #ac3223, /* gradient */
+    1px 1px 3px rgba(0, 0, 0, .75); /* shadow */
+    text-shadow: 0 1px 2px rgba(0, 0, 0, .75);
+
+    font-size: 14px;
+
+    &:hover {
+        background: #a61715;
+        box-shadow: inset 1px 1px 0 rgba(255, 255, 255, .25), /* highlight */
+        inset 0 0 6px #da3b2c, /* inner glow */
+        inset 0 80px 80px -40px #dd4330, /* gradient */
+        1px 1px 3px rgba(0, 0, 0, .75); /* shadow */
+        text-shadow: 0 1px 2px rgba(0, 0, 0, .75), 0 0 40px #ffffff;
+    }
+    &.button__add {
```

Comment:
It's ok, but usually not simply one element located inside block
```
&.button {
  &__add {

  }
}
```

Reply:
According to the lots of code above this line, it looks more like modifier, not (child) element. You should use `--` instead of `__`, i. e. `&--add` as @AleshaOleg wrote before me.

Reply:
Perhaps I'm still wrong, but I re-read the BEM documentation in it that the element name is separated from the block name by two underscores (__).
Is it really necessary to change?
https://ru.bem.info/methodology/quick-start/#%D0%9C%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80